### PR TITLE
Chrome dev beta support

### PR DESCRIPTION
**Pull request by ruescar**

### CODE REVIEW

File: selenium/automate_chrome.sh [hunk 1]
```diff
@@ -3,28 +3,54 @@ set -e
 input=$1
 driver_version=$2
 tag=$3
+channel=${4:-"stable"}
 
 if [ -z "$1" -o -z "$2" -o -z "$3" ]; then
-    echo 'Usage: ./automate_chrome.sh <browser_version|package_file> <chromedriver_version> <tag_version>'
+    echo 'Usage: ./automate_chrome.sh <browser_version|package_file> <chromedriver_version> <tag_version> [{stable|dev|beta}]'
```
Comment (syerrapragada) [inline selenium/automate_chrome.sh]: Do we need another parameter [{stable|dev|beta}] for tagging?

File: selenium/automate_chrome.sh [hunk 2]
```diff
@@ -3,28 +3,54 @@ set -e
...
     exit 1
 fi
 set -x
 
 browser_version=$input
 method="chrome/apt"
 if [ -f "$input" ]; then
-    cp "$input" chrome/local/google-chrome-stable.deb
```
Comment (syerrapragada) [inline selenium/automate_chrome.sh]: Let's keep the changes to minimal. Do we need to change this line?

File: selenium/automate_chrome.sh [hunk 3]
```diff
@@ -3,28 +3,54 @@ set -e
...
+    cp "$input" chrome/local/google-chrome.deb
     filename=$(echo "$input" | awk -F '/' '{print $NF}')
     browser_version=$(echo $filename | awk -F '_' '{print $2}' | awk -F '-' '{print $1}')
     method="chrome/local"
 fi
+if [ "$browser_version" == "latest" ]; then
```
Comment (syerrapragada) [inline selenium/automate_chrome.sh]: Instead of `latest`, please use `stable`?

**Review by syerrapragada — CHANGES_REQUESTED** [review]

File: selenium/automate_chrome.sh (hunk 1 above)
Comment (ruescar) [reply selenium/automate_chrome.sh]: Changed

File: selenium/automate_chrome.sh (hunk 2 above)
Comment (ruescar) [reply selenium/automate_chrome.sh]: Reverted

File: selenium/automate_chrome.sh (hunk 3 above)
Comment (ruescar) [reply selenium/automate_chrome.sh]: Refactored

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+          command: |
+            echo "Browser: ${BROWSER_NAME}"
+            echo "Channel: ${BROWSER_CHANNEL}"
+            docker inspect selenoid/${BROWSER_NAME}:${BROWSER_CHANNEL} | jq -r '.[0].Config.Labels'
+      - run:
+          name: Checkout tests
+          command: cd .. && git clone -b ${TESTS_BRANCH} --single-branch git@github.com:twilio/selenoid-container-tests.git
+      - run:
+          name: Test image
+          command: cd ../selenoid-container-tests && mvn clean test -Dgrid.connection.url="http://localhost:4445/" -Dgrid.browser.name=${BROWSER_NAME} -Dgrid.browser.version=${BROWSER_CHANNEL}
```
Comment (syerrapragada) [inline .circleci/config.yml]: Add a new line at the end of the file.

File: README.md [hunk 5]
```diff
@@ -43,8 +43,26 @@ Here `63.0+build2-0ubuntu0.16.04.2` is `firefox` package version for Ubuntu 16.0
...
+$ ./automate_chrome.sh 70.0.3538.67-1 2.43 stable
+$ ./automate_chrome.sh 74.0.3729.40-1 2.43 beta
+$ ./automate_chrome.sh 75.0.3745.4-1 2.43 dev
+$ ./automate_chrome.sh google-chrome-stable_70.0.3538.67-1_amd64.deb 2.43 70.0
+$ ./automate_chrome.sh google-chrome-stable_70.0.3538.67-1_amd64.deb 2.43 stable
+$ ./automate_chrome.sh google-chrome-beta_74.0.3729.40-1_amd64.deb 2.43 beta
+$ ./automate_chrome.sh google-chrome-unstable_75.0.3745.4-1_amd64.deb 2.43 dev
 ```
 Here `70.0.3538.67-1` is `google-chrome-stable` package version for Ubuntu 16.04, `2.43` is [Chromedriver](https://chromedriver.storage.googleapis.com/index.html) version, `70.0` is Docker tag to be applied.  
+`Note` that if Chrome is from a a different channel to `stable`, Docker tag must be `{beta|dev}`
```
Comment (syerrapragada) [inline README.md]: a a -> a

File: selenium/build-dev.sh [hunk 6]
```diff
@@ -38,13 +42,31 @@ if [ "$browser" == "chrome/local" -o "$browser" == "opera/blink/local" -o "$brow
     cp $debWildcard "$dir_name"
     docker rm -f static-server || true
     docker run -d -p 8080:8043 -v "$dir_name":/srv/http --name static-server pierrezemb/gostatic
-    host_ip=$(ifconfig docker0 | grep inet | grep -v inet6 | awk '{print $2;}' | sed -e 's|addr:||g')
+    host_ip=$(ifconfig | grep -E "([0-9]{1,3}\.){3}[0-9]{1,3}" | grep -v 127.0.0.1 | awk '{ print $2 }' | cut -f2 -d: | head -n1)
```
Comment (syerrapragada) [inline selenium/build-dev.sh]: Could we take this approach if `docker0` is not found?

**Review by syerrapragada — COMMENTED** [review]
LGTM but left a few questions for you

**Review by syerrapragada — APPROVED** [review]

File: .circleci/config.yml (hunk 4 above)
Comment (ruescar) [reply .circleci/config.yml]: Fixed

File: README.md (hunk 5 above)
Comment (ruescar) [reply README.md]: Fixed

File: selenium/build-dev.sh (hunk 6 above)
Comment (ruescar) [reply selenium/build-dev.sh]: Fixed

**Review by syerrapragada — APPROVED** [review]

**Pull request merged**